### DISCUSSION TITLE
Unable to run your project. The "RunCommand" property is not defined

### DISCUSSION THREAD

**Discussion by taipignas** [aws/aws-lambda-dotnet#1291] [Q&A] [answered]
### Describe the bug

I am not able to debug the project. I get this error.

![image](https://user-images.githubusercontent.com/55182635/187138962-dcc69149-c7b1-4289-9c1f-fb7c81e4ebd6.png)

### Expected Behavior

debugging should work

### Current Behavior

debugging doesnt work

### Reproduction Steps

1. create new annotation framework serverless application
2. run

### AWS .NET SDK and/or Package version used

AWSToolkitPackage.v17

### Targeted .NET Platform

.NET 6

### Operating System and version

Windows 10

**Comment by ashishdhingra** [comment, marked as answer]
Hi @taipignas,

Good morning.

The reason you are getting this error because Serverless annotations project is a class library project which cannot be executed directly. In order to test/debug annotations project locally, you need to have [Mock Lambda Test Tool](https://github.com/aws/aws-lambda-dotnet/tree/master/Tools/LambdaTestTool) installed and it needs to be selected when running the project. In Lambda test tool, you could then select `API Gateway V2 HTTP API` from example requests dropdown to test your API.

Thanks,
Ashish
